multi_cgra/test/MeshMultiCgraTemplateRTL_test.py

```diff
@@ -152,16 +152,18 @@ def run_sim(test_harness, max_cycles = 200):
   test_harness.sim_tick()
 
 
-def test_mesh_multi_cgra_universal(cmdline_opts):
-  arch_file = os.path.join(os.path.dirname(__file__), "arch.yaml")
-  parser = Parser(arch_file)
-  multiCgraParam = parser.parse_multi_cgra_param()
+def test_mesh_multi_cgra_universal(cmdline_opts, multiCgraParam = None):
```

Comment:
Instead of `multiCgraParam`, can we have:
```
def test_mesh_multi_cgra_universal(cmdline_opts, arch_yaml_path = "arch.yaml"):
```

I think this would be easier to use, user no need to parse the yaml before calling this func then.